### PR TITLE
 Apply black style to more test_SeqIO* files , version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to the below files; standard black style changes.
test_SeqIO_online.py
test_SeqIO_NibIO.py
test_SeqIO_Insdc.py
test_SeqIO_features.py

### CODE REVIEW

File: Tests/test_SeqIO_features.py [hunk 1]
```diff
@@ -80,16 +87,22 @@ def compare_record(old, new, expect_minor_diffs=False):
             continue
         if key == "molecule_type":
             # EMBL allows e.g. "genomics DNA" where GenBank is limited.
-            common_words = set(old.annotations[key].split()).intersection(new.annotations[key].split())
+            common_words = set(old.annotations[key].split()).intersection(
+                new.annotations[key].split()
+            )
```
Comment (peterjc) [inline Tests/test_SeqIO_features.py]: This doesn't look so nice... the one line version must be right on the 88 limit. Perhaps two lines?

```python
common_words = set(old.annotations[key].split())
common_words = common_words.intersection(new.annotations[key].split())
```

**Comment by peterjc** [comment]
The flake8 problem was fixed on the master branch, we can ignore this:

```
$ flake8
./Bio/SearchIO/_model/hit.py:227:31: BLK100 Black would make changes.
./Bio/SearchIO/_model/hsp.py:17:33: BLK100 Black would make changes.
```

File: Tests/test_SeqIO_features.py (hunk 1 above)
Comment (svalqui) [reply Tests/test_SeqIO_features.py]: amended, Thank you.

**Comment by svalqui** [comment]
Thank you for reviewing this.

**Pull request merged**